src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -354,30 +364,19 @@ private JsonObject formatMessageAsJson(long timestamp, String logLevelName, Stri
         logMessage.addProperty("logger", loggerName);
         logMessage.addProperty("thread", threadName);
 
-        if (throwableProxy.isPresent()) {
-            logMessage.addProperty("exception", formatThrowableProxy(throwableProxy.get()));
+        if (loggingEvent.isPresent()) {
+            if (loggingEvent.get().getThrowableProxy() != null) {
+                logMessage.addProperty("exception", throwableProxyConverter.convert(loggingEvent.get()));
+            }
+        }
+
+        if (mdcPropertyMap.isPresent()) {
```

Comment:
Do this first thing and comment it so people will understand. You don't want the major field to be overridden, even by mistake, with the MDC field values.